app/src/main/java/com/github/se/orator/ui/settings/SettingsScreen.kt

```diff
@@ -45,66 +43,22 @@ import com.github.se.orator.ui.theme.AppFontSizes
 data class SettingBar(
     val text: String,
     val testTag: String,
-    val function: (AppThemeViewModel?) -> Unit,
     val icon: ImageVector,
     val iconDescription: String
 )
 
 // creating a list of all settings to implement them with a simple loop
 val listOfSettings =
     listOf(
-        SettingBar(
-            "Account Management",
-            "account_management",
-            { Log.d("hello", "account management") },
-            Icons.Outlined.AccountCircle,
-            "account circle icon"),
-        SettingBar(
-            "Storage Settings",
-            "storage_settings",
-            { Log.d("hello", "storage settings") },
-            Icons.AutoMirrored.Outlined.List,
-            "storage settings icon"),
-        SettingBar(
-            "Permissions",
-            "permissions",
-            { Log.d("hello", "permissions") },
-            Icons.Outlined.Lock,
-            "lock icon"),
-        SettingBar(
-            "Theme (click to switch)",
-            "theme",
-            { themeVM ->
-              themeVM?.switchTheme()
-              Log.d("hello", "theme")
-            },
-            Icons.Outlined.DarkMode,
-            "theme"),
-        SettingBar(
-            "Invite Friends",
-            "invite_friends",
-            { Log.d("hello", "invite friends") },
-            Icons.Outlined.Mail,
-            "mail icon"),
-        SettingBar(
-            "Notifications",
-            "notifications",
-            { Log.d("hello", "Notifications") },
-            Icons.Outlined.Notifications,
-            "notifications icon"),
-        SettingBar(
-            "Rate on the App Store",
-            "rate_on_the_app_store",
-            { Log.d("hello", "rate on the app store") },
-            Icons.Outlined.Star,
-            "star icon"),
-        SettingBar("About", "about", { Log.d("hello", "about") }, Icons.Outlined.Info, "info icon"))
+        SettingBar("Permissions", "permissions", Icons.Outlined.Lock, "lock icon"),
+        SettingBar("Theme", "theme", Icons.Outlined.DarkMode, "theme"),
+        SettingBar("About", "about", Icons.Outlined.Info, "info icon"))
 
 // reusable function that is called to add a section to settings
 @Composable
-fun TextButtonFun(settingBar: SettingBar, switchTheme: AppThemeViewModel? = null) {
+fun TextButtonFun(settingBar: SettingBar, onClick: () -> Unit = {}) {
```

Comment:
It's great to see you opting for a generic `onClick` function instead of relying on a default null value!